tests/postgres_tests/test_indexes.py

```diff
@@ -455,3 +466,90 @@ def test_spgist_parameters(self):
         with connection.schema_editor() as editor:
             editor.remove_index(CharFieldModel, index)
         self.assertNotIn(index_name, self.get_constraints(CharFieldModel._meta.db_table))
+
+    def test_func_index_with_jsonb_key_transform(self):
+        """
+        Test creating a B-Tree index on key in a JSONB-field
```

Comment:
To add more explanation, docstrings should state the expected behavior and omit prefixes like "Ensure that"/"Test" since all tests test things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

Reply:
I just removed the docstrings, they where quite redundant by the method names.